src/graph/01_bfs.md

```diff
@@ -17,27 +17,44 @@ In this article we demonstrate how we can use BFS to solve the SSSP (single-sour
 We can develop the algorithm by closely studying Dijkstra's algorithm and thinking about the consequences that our special graph implies.
 The general form of Dijkstra's algorithm is (here a `set` is used for the priority queue):
 
-```cpp
-d.assign(n, INF);
-d[s] = 0;
-set<pair<int, int>> q;
-q.insert({0, s});
-while (!q.empty()) {
-    int v = q.begin()->second;
-    q.erase(q.begin());
-
-    for (auto edge : adj[v]) {
-        int u = edge.first;
-        int w = edge.second;
-
-        if (d[v] + w < d[u]) {
-            q.erase({d[u], u});
-            d[u] = d[v] + w;
-            q.insert({d[u], u});
+=== "C++"
+    ```cpp
+    d.assign(n, INF);
+    d[s] = 0;
+    set<pair<int, int>> q;
+    q.insert({0, s});
+    while (!q.empty()) {
+        int v = q.begin()->second;
+        q.erase(q.begin());
+
+        for (auto edge : adj[v]) {
+            int u = edge.first;
+            int w = edge.second;
+
+            if (d[v] + w < d[u]) {
+                q.erase({d[u], u});
+                d[u] = d[v] + w;
+                q.insert({d[u], u});
+            }
         }
     }
```

Comment:
```suggestion
=== "C++"
    ```cpp
    d.assign(n, INF);
    d[s] = 0;
    priority_queue<pair<int, int>> q = {{0, s}};
    while (!empty(q)) {
        int [dv, v] = q.top();
        q.pop();
        if (-dv == d[v]) {
          for (auto [u, w] : adj[v]) {
              if (d[v] + w < d[u]) {
                  d[u] = d[v] + w;
                  q.insert({-d[u], u});
              }
          }
        }
    }
```
Probably best to make this consistent with Python then. Also remove the comment about "set" above if this is incorporated.